### ISSUE TITLE
Import Pages - csv example file improvement

### ISSUE THREAD

**Issue by MSoeb** [OpenSlides/openslides-client#3822]
**Description:** Some improvements of the account and participant import csv example files are required. 

**What should be done:**

- [x] Participant import: Only one example line is needed. The csv content should look like the following image:
![image](https://github.com/OpenSlides/openslides-client/assets/105491030/2604d1ea-1886-4237-b50e-61df03e36d45)

- [x] Account import: Only one example line is needed. The csv content should look like the following image:
![image](https://github.com/OpenSlides/openslides-client/assets/105491030/f45b1bb7-9bc6-441f-9e40-d18f7c9f7fe0)

- <s>If possible: Values in groups and gender should be translated in the csv example file when switching languages in the OpenSlides client. This does NOT affect the column name, only the content entered. (Example: male -> männlich, Delegates -> Delegierte). This would make it much easier for users to recognize, that they could use terms of their chosen OpenSlides client language.</s>

Info: This is part of the META Issue #3809

**Comment by bastianjoel** [comment]
> If possible: Values in groups and gender should be translated in the csv example file when switching languages in the OpenSlides client. This does NOT affect the column name, only the content entered. (Example: male -> männlich, Delegates -> Delegierte). This would make it much easier for users to recognize, that they could use terms of their chosen OpenSlides client language.

No, this not possible. This will have **massive** side effects.
